src/staging/src/components/pageContent/linkCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import PropTypes from "prop-types";
+import { Link } from "gatsby"
+import "../../styles/gridCard.scss"
+
+
+export default function LinkCard({ url, title, imageUrl, imageAltText }) {
+  return (
+    <Link
+      className="card grid-card grid-card--horz grid-card--light"
+      to={url}
+    >
+      <div className="card-img">
+        <img src={imageUrl} alt={imageAltText ?? null} />
+      </div>
+      <div className="card-body">
+        <h5 className="card-body-header">{title}</h5>
+        <div className="card-button">
+          Learn more <i className="fa fa-chevron-circle-right"></i>
```

Comment:
Should we make "learn more" a param? Possibly the icon as well? Maybe for a future iteration

Reply:
Good question. I was thinking about this but it didn't seem necessary yet.